build.rs

```diff
@@ -151,10 +152,10 @@ fn main() {
         pkg_check("flex");
         pkg_check("bison");
         pkg_check("gawk");
+        pkg_check("aclocal");
```

Comment:
What needs `aclocal`?

Reply:
When configuring the build of elfutils on my system, it was indeed used, so I added it.